src/ff7/menu.cpp

```diff
@@ -133,3 +134,27 @@ uint32_t ff7_menu_decrease_item_quantity(uint32_t item_used)
     g_FF7SteamAchievements->unlockLastLimitBreakAchievement(item_used & 0x1FF);
     return item_id & 0xFFFF0000 | (uint32_t)local_c;
 }
+
+void dispatchAttackCommand(){
+    *ff7_externals.issued_command_id = 0x01;
+    *ff7_externals.issued_action_target_type = 0;
+    *ff7_externals.issued_action_target_index = 4;
+    ((void(*)())ff7_externals.dispatch_chosen_battle_action)();
+}
+
+void ff7_battle_menu_sub_6DB0EE(){
+    ((void(*)())ff7_externals.battle_sub_6DB0EE)();
+    if(gamehacks.isAutoAttack() && (*ff7_externals.battle_menu_state >= 0 && *ff7_externals.battle_menu_state < 19)){

```

Comment:
Don't really know if the battle menu state from 0 to 19 (excluded) covers all possible battle menu of the active character. I took this value from SisterRay (https://github.com/FFT-Hackers/SisterRay/blob/master/include/SisterRay/constants.h#L158), and confirmed the 19 in FF7.exe code